blog/migrations/0002_remove_post_posts_post_author.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.1.7 on 2023-06-01 13:23
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blog', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='post',
+            name='posts',
+        ),
+        migrations.AddField(
+            model_name='post',
+            name='author',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='posts', to='blog.author'),
+        ),
```

Comment:
Это просто пример, так делать не будем
```suggestion
        migrations. RenameField(model_name='post', 'posts', 'author'),
```